community/io/src/test/java/org/neo4j/graphdb/mockfs/EphemeralFileSystemAbstraction.java

```diff
@@ -1273,16 +1224,14 @@ private void verifySize( int totalAmount )
 
             // Double size each time, but after 1M only increase by 1M at a time, until required amount is reached.
             int newSize = buf.capacity();
-            int sizeIndex = sizeIndexFor( newSize );
-            while ( capacity( sizeIndex ) < totalAmount )
```

Comment:
The comment is now out of date. It might not be a bad idea to restrict the doubling a bit.